hack/non-olm-install/run-e2e-kuttl-test.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+export NON_OlM="true"
```

Comment:
`l` is in lower case. Convert to all upper case `NON_OLM`.

Reply:
Done